docs/containers-registries.conf.5.md

```diff
@@ -102,6 +102,70 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Short-Name Aliasing
+The use of unqualified-search registries entails an ambiguity as it's
+unclear from which registry a given image, referenced by a short name,
+may be pulled from.
+
+As mentioned in the note at the end of the document, using short names is
+subject to the risk of spoofed registry namespaces.  If the unqualified-search
+registries are set to `["registry1.com", "registry2.com"]` an attacker may take
+over a namespace of registry1.com such that an image may be pulled from
+registry1.com instead of the intended soruce registry2.com.
+
+While it is highly recommended to always use fully-qualified image references,
+existing deployments using short names may not be easily changed.  To
+circumvent the aforementioned ambiguity, so called short-name aliases can be
+configured that allow to alias a short name to a fully-qualified image
+reference.
+
+Short-name aliases can be configured in the `[aliases]` table in the form of
+`"name"="value"` with the left-hand `name` being the short name (e.g., "image")
+and the right-hand `value` being the fully-qualified image reference (e.g.,
+"registry.com/namespace/image").  Note that neither "name" nor "value" can
+include a tag or digest.  Moreover, "name" must be a short name and hence
+cannot include a registry domain or refer to localhost.
```

Comment:
Do we guard against duplicate aliases entered into the table?  Which one do we use if they accidentally add one and do we need to mention anything here about that?

Reply:
maybe I'll discover it later, but we have `quay.io/buildah/stable:latest :v1.16.0` and other  tags.  Does the alias always pull latest?  Is there a way to specify a particular tag?  i.e.  `fedora:31` vs `fedora:32`

Reply:
I'll clarify.

Duplicate entries will cause a TOML load error. Tags and digests are not allowed in any config.